doc/architecture/diagrams/data-structures/programmes-organisations-people-referrals.wsd

```diff
@@ -0,0 +1,90 @@
+@startuml
```

Comment:
Is it worth documenting how to use this file? Does it need a StarUML licence?

Reply:
It doesn't, we can copy the code into something like PlantUML to tweak it, that's what we used to generate the diagrams. Would documenting this in the commit message be enough, do you think?

Reply:
I think that's probably fine, unless other devs are likely to need to get involved in that, in which case it might be nice to add to `/doc` (but I doubt that's the case)